olp-cpp-sdk-dataservice-read/src/generated/model/SubscribeResponse.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#pragma once
+
+#include <string>
+
+#include <boost/optional.hpp>
+
+namespace olp {
+namespace dataservice {
+namespace read {
+namespace model {
+
+class SubscribeResponse final {
+ public:
+  const boost::optional<std::string>& GetNodeBaseURL() const {
+    return node_base_url_;
+  }
+  void SetNodeBaseURL(std::string value) { node_base_url_ = std::move(value); }
+
+  const boost::optional<std::string>& GetSubscriptionId() const {
+    return subscription_id_;
+  }
+  void SetSubscriptionId(std::string value) {
+    subscription_id_ = std::move(value);
+  }
+
+  const boost::optional<std::string>& GetCorrelationId() const {
+    return x_correlation_id;
+  }
+  void SetCorrelationId(std::string value) {
+    x_correlation_id = std::move(value);
+  }
+
+ private:
+  /// A subscription/Kafka Consumer is created in a specific server instance.
+  /// This base URL represents the specific server node on which this
+  /// subscription/Kafka Consumer was created. This base URL should be used
+  /// instead of the base URL returned by the `lookup` service when making
+  /// requests to these endpoints: /partitions, /offsets, /seek and /subscribe
+  /// (DELETE).
+  boost::optional<std::string> node_base_url_;
```

Comment:
It doesn't seem optional to me. Can we do anything without this node_base_url_?
I think not. If it is empty which URL should we use?

Reply:
Logically you're right. But this field is not marked as mandatory in OLP API.
Should I remove optional for it?

Reply:
Yes, please.